independent-publisher-connectors/ShipStationIP/apiDefinition.swagger.json

```diff
@@ -372,10 +372,13 @@
     }
   },
   "securityDefinitions": {
-    "auth": {
+    "basic_auth": {
       "type": "basic"
     }
   },
+  "security": {

```

Comment:
@SmolSoftBoi one of validation check fails because value for this property is an array of objects (one object in our case which contains "basic_auth" property).
Please refer to [this example](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/independent-publisher-connectors/Udemy/apiDefinition.swagger.json#L808).

Could you please make one last fix?